### PR TITLE
WIP update about

### PR DESCRIPTION
**Pull request by mcfiredrill**
Updating the about section with some of the new values that I think we've come to adopt.

Please comment if you have any feedback!

### CODE REVIEW

**Comment by dudethatbe** [comment]
who's even heard of a `netradio`?

**Comment by mcfiredrill** [comment]
Need some translations for this....

**Comment by mcfiredrill** [comment]
Got a ja translation from Aka Obi! https://twitter.com/aka_obi/status/1286835779738537984

File: translations/ko.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "thanks": "감사합니다!"
   },
   "just_a_website": "그냥 웹 사이트 일뿐입니다.",
-  "about_datafruits": "Datafruits는 Tony Miller (mcfiredrill)가 2012 년에 설립 한 netradio 및 netlabel입니다. 그 임무는 요를 가져 오는 것입니다.세계에서 가장 이상한 전자 음악과 전 세계의 전자 음악 공동체를 만날 수 있습니다.",
+  "about_datafruits": "국제적인 음악 커뮤니티를 위한 국제적인 음악 커뮤니티에 의한 인디펜던트 라디오.datafruits는 오픈 소스이며, 다양한 경계와 장르를 파괴한다.우리는 어떤 곡이든 쓸 수 있어요.서비스가 미흡해 주변에 내몰리는 커뮤니티를 재조명하고 이들로부터 발신되는 음악을 나누는 데 수고를 아끼지 않는다.사용자에게 도움이 되는 플랫폼은 그 플랫폼의 이용자와 DJ와 청취자에 의해 소유되고 운영되는 것과 다름없다고 우리는 생각합니다!",
```
Comment (mcfiredrill) [inline translations/ko.json:50]: I ran this through papago...do you think you could correct this one ? @fruitskikizin

**Comment by mcfiredrill** [comment]
Just merging for now, will do some edits later.

**Pull request merged**